models/dirwatcher.js

```diff
@@ -0,0 +1,24 @@
+import Promise from 'bluebird';
+import EventEmitter from 'events';
+import fs from 'fs';
+import Log from 'log';
+
+const log = new Log();
+const readdirAsync = Promise.promisify(fs.readdir);
+
+export class DirWatcher extends EventEmitter {
```

Comment:
It doesn't check adding new files. But it's the main use case (at least I think so :))

Reply:
Changes have been made to use 'chokidar'. For now it does